keras2onnx/ke2onnx/main.py

```diff
@@ -69,6 +69,28 @@ def convert_keras_concat(scope, operator, container):
                  operator_name=operator.full_name, axis=axis)
 
 
+def convert_keras_flatten(scope, operator, container):
```

Comment:
Why not use the [ONNX Flatten Op](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Flatten)?